src/main/java/org/mockito/Mockito.java

```diff
@@ -2568,4 +2568,13 @@ public static VerificationMode description(String description) {
     static MockitoDebugger debug() {
         return new MockitoDebuggerImpl();
     }
+
+    /**
+     * Set the verification strategy (direct or lazily) when invoking {@link #verify(Object)}.
+     *
+     * @param verificationState The type of verification.
+     */
+    public static void setVerificationState(VerificationStrategy verificationState) {
```

Comment:
-1 to this new public method.

I am very cautious about introducing new public methods to the main entry point API. In this particular instance, I'm not convinced we need it :)